pkg/proc/linutil/regs_arm64_arch.go

```diff
@@ -130,7 +130,23 @@ func (r *ARM64Registers) Copy() proc.Registers {
 	return &rr
 }
 
-// Decode decodes an XSAVE area to a list of name/value pairs of registers.
+type ARM64PtraceFpRegs struct {
+	vregs [512]byte
+	fpsr  uint32
+	fpcr  uint32
+}
+
+func (fpregs *ARM64PtraceFpRegs) Decode() (regs []proc.Register) {
+	for i := 0; i < len(fpregs.vregs); i += 16 {
+		regs = proc.AppendFPReg(regs, fmt.Sprintf("V%d", i/16), fpregs.vregs[i:i+16])
+	}
+	return
+}
+
+func (fpregs *ARM64PtraceFpRegs) Byte() []byte {
+	return fpregs.vregs[:]
+}
+
 func Decode(fpregs []byte) (regs []proc.Register) {
```

Comment:
Why left two `Decode()`?

Reply:
We cannot use/modify already defined Decode API 

> func Decode(fpregs []byte) (regs []proc.Register) 

as we need to use `ARM64PtraceFpRegs` here so I created new

> func (fpregs *ARM64PtraceFpRegs) Decode() (regs []proc.Register)